desktop/lib/messagebundles/pt.po

```diff
@@ -1415,118 +1412,105 @@ msgstr "Completo"
 #. time elapsed, time left, download speed
 #: /Users/gubatron/workspace.frostwire/frostwire/desktop/src/com/frostwire/gui/components/transfers/TransferDetailGeneral.java:100
 msgid "Time elapsed"
-msgstr ""
+msgstr "Tempo decorrido"
 
 #: /Users/gubatron/workspace.frostwire/frostwire/desktop/src/com/frostwire/gui/components/transfers/TransferDetailGeneral.java:102
-#, fuzzy
 msgid "Time left"
-msgstr "Tempo"
+msgstr "Tempo restante"
 
 #: /Users/gubatron/workspace.frostwire/frostwire/desktop/src/com/frostwire/gui/components/transfers/TransferDetailGeneral.java:104
-#, fuzzy
 msgid "Download speed"
-msgstr "Velocidade de download:"
+msgstr "Velocidade de download"
 
 #: /Users/gubatron/workspace.frostwire/frostwire/desktop/src/com/frostwire/gui/components/transfers/TransferDetailGeneral.java:112
-#, fuzzy
 msgid "Download speed limit"
-msgstr "Velocidade de download:"
+msgstr "Limite de velocidade de download"
 
 #: /Users/gubatron/workspace.frostwire/frostwire/desktop/src/com/frostwire/gui/components/transfers/TransferDetailGeneral.java:121
-#, fuzzy
 msgid "Upload speed"
-msgstr "Velocidade de upload:"
+msgstr "Velocidade de upload"
 
 #. Total Size, Peers, Upload speed limit
 #. Share Ratio
 #: /Users/gubatron/workspace.frostwire/frostwire/desktop/src/com/frostwire/gui/components/transfers/TransferDetailGeneral.java:126
-#, fuzzy
 msgid "Total size"
-msgstr "Upstream Total:"
+msgstr "Tamanho total"
 
 #: /Users/gubatron/workspace.frostwire/frostwire/desktop/src/com/frostwire/gui/components/transfers/TransferDetailGeneral.java:130
-#, fuzzy
 msgid "Upload speed limit"
-msgstr "Velocidade de upload:"
+msgstr "Limite de velocidade de upload"
 
 #. TODO: Add settings_gray button and dialog to adjust upload speed limit
 #: /Users/gubatron/workspace.frostwire/frostwire/desktop/src/com/frostwire/gui/components/transfers/TransferDetailGeneral.java:133
-#, fuzzy
 msgid "Share ratio"
-msgstr "Compartilhar Taxa"
+msgstr "Proporção de ações"
 
 #: /Users/gubatron/workspace.frostwire/frostwire/desktop/src/com/frostwire/gui/components/transfers/TransferDetailGeneral.java:146
-#, fuzzy
 msgid "Save location"
-msgstr "Compartilhar Taxa"
+msgstr "Localização no disco"
 
 #: /Users/gubatron/workspace.frostwire/frostwire/desktop/src/com/frostwire/gui/components/transfers/TransferDetailGeneral.java:152
-#, fuzzy
 msgid "InfoHash"
-msgstr "Copiar Infohash"
+msgstr "Infohash"
 
 #: /Users/gubatron/workspace.frostwire/frostwire/desktop/src/com/frostwire/gui/components/transfers/TransferDetailGeneral.java:156
 msgid "Magnet URL"
-msgstr ""
+msgstr "URL magnético"
```

Comment:
I think this is better left as "Magnet URL" or "URL de Magnet"

Reply:
Look https://www.tecmundo.com.br/torrent/4387-o-que-e-link-magnetico-.htm :)

Reply:
I just think it's like translating "Torrent" or "Download" which we don't.
I'm also not sure of the tech pro-efficiency of that writer.

Reply:
It should also be capitalized as "Magnet".

And another point to my side, it's not a "Magnetic Link" in English either, which would be correct. It's a "Magnet"

Reply:
if anything in spanish/portuguese the translation would be "URL de Magneto"